recordset/recordset.html

```diff
@@ -32,6 +32,17 @@ <h2 id="page-title">Records for {{ vm.tableDisplayName }}</h2>
                 <button class="btn btn-sm btn-default dropdown-toggle" type="button" data-toggle="dropdown">Items per page
                     <span class="caret"></span></button>
                 <ul class="dropdown-menu">
+                    <li ng-if="pageInfo.pageLimit !== 10 &&
+                        pageInfo.pageLimit !== 10 &&
+                        pageInfo.pageLimit !== 25 &&
+                        pageInfo.pageLimit !== 50 &&
+                        pageInfo.pageLimit !== 75 &&
+                        pageInfo.pageLimit !== 100 &&
+                        pageInfo.pageLimit !== 200"><a id="custom-page-size" href ng-click="pageLimit(pageInfo.pageLimit)">
```

Comment:
Is this supposed to check if the user defined a custom page size? Maybe this check should be moved to a function even though it is only being used once.